src/modules/verifyCardNumber/index.ts

```diff
@@ -20,11 +19,13 @@ function verifyCardNumber(digits: number): boolean | undefined {
 		return false;
 	}
 
-	let sum = 0;
-	let even, subDigit;
+	let oddOrEven,
+		subDigit,
+		sum = 0;
+
 	for (let i = 0; i < 16; i++) {
-		even = i % 2 === 0 ? 2 : 1;
-		subDigit = parseInt(digitsResult.substr(i, 1), 10) * even;
+		oddOrEven = i % 2 === 0 ? 2 : 1;
```

Comment:
I think you are in the right way and you understood this function as well, and I appreciate your attention. So this line is the Radix of every sub digits and I suggest you rename the `oddOrEven` var name to `radix`.